src/content/whats-new/2024/04/whats-new-04-09-eol-synthetics-runtime-cpm.md

```diff
@@ -0,0 +1,36 @@
+---
+title: 'Upgrade to the new synthetic monitor runtime to prevent impacts to your synthetic monitors'
+summary: 'Support for legacy synthetics runtimes and Containerized Private Minion will be concluded on October 9, 2024'
+releaseDate: '2024-02-21'
+learnMoreLink: 'https://docs.newrelic.com/docs/synthetics/synthetic-monitoring/using-monitors/runtime-upgrade-ui'
+getStartedLink: ''
+---
+
+On October 9, 2024, New Relic will end-of-life (EOL) legacy synthetics runtimes including our legacy Chrome 72 (and older) and Node 10 (and older) Synthetics runtimes and the Containerized Private Minion (CPM). Please refer to our [transition guide](https://docs.newrelic.com/docs/synthetics/synthetic-monitoring/using-monitors/new-runtime/) to update to the latest runtimes or contact support. 
+
+The new runtimes provide many customer benefits that cannot be replicated in the legacy runtime environment, including but not limited to: 
+- The elimination of security CVE findings that cannot be resolved due to the legacy runtime design and backward compatibility requirements of the Containerized Private Minion (CPM)
+- Support for newer Chrome and Node versions, including easier upgrade paths
```

Comment:
AFAICT people do refer to Node.js as just Node, but I wonder if we should be more formal about it. Google Cloud uses "Node.js runtime" and the node.js docs also refer to it as the Node.js runtime.

Reply:
I actually can't tell lol. I looked at "node v10" on their page and they say Node v10... Maybe it's ok